src/pages/ServiceList/ServiceListComponent.tsx

```diff
@@ -7,39 +7,61 @@ import { ActiveFilter, FilterType } from '../../types/NamespaceFilter';
 import * as API from '../../services/Api';
 import Namespace from '../../types/Namespace';
 import { Pagination } from '../../types/Pagination';
-import { IstioLogo, ServiceItem, ServiceList, SortField } from '../../types/ServiceListComponent';
+import { IstioLogo, ServiceItem, ServiceList, SortField, overviewToItem } from '../../types/ServiceListComponent';
 import PropTypes from 'prop-types';
 import MetricsOptionsBar from '../../components/MetricsOptions/MetricsOptionsBar';
+import { getRequestErrorsRatio } from '../../components/ServiceHealth/HealthHelper';
 import { HealthIndicator, DisplayMode } from '../../components/ServiceHealth/HealthIndicator';
 import ServiceErrorRate from './ServiceErrorRate';
-import ServiceItemComparer from './ServiceItemComparer';
 import RateIntervalToolbarItem from './RateIntervalToolbarItem';
 
 import './ServiceListComponent.css';
 
-const sortFields: SortField[] = [
+export const sortFields: SortField[] = [
```

Comment:
I don't like to have mixed criteria for export functions.
I think this object should be uppercase, Alberto started that and I don't see bad to follow that rule.
Having lowercase and uppercase exports in methods are confusing.
I would prefer to use a single criteria, and just to follow the preliminar, majority one.

Reply:
sure .. actually this is only exported to be accessible for test, let me see if I can change the test and not export it at all

Reply:
I wonder if we should start to use something to test non-exported stuff (like "rewire", see https://stackoverflow.com/questions/14874208/how-to-access-and-test-an-internal-non-exports-function-in-a-node-js-module )

It's a shame we have to export functions in order to test them. What do you think?

cc @mtho11 / @vnugent

Reply:
@lucasponce yes I fully understand your point (+100 on having unified rules, I already asked for this in the past but it didn't got much attention) and I agree not to leave my change as is.

But I'm stepping to another point about testing non-exported stuff because it's also the subject here. I would actually prefer not to export things that are not meant to be accessed from other modules. I've just discovered rewire now, I also see here are [other suggestions](https://stackoverflow.com/a/32071275/3697695) for similar problem.

Reply:
By the way, in API.ts I don't really understand why it's upper camel case. It's the only place where we have it. Just have a search of "export const" in the project: almost everything is low camel case, which is the most common in js.

Reply:
That's the point I tried to spot.

I guess the rule should be:
- Let's follow what it is in the code, as someone introduced first.
- If a style should be changed, some discussion, and a global change.
- But to introduce a partial change is not good, as it grows two different styles, and it creates confusion.
- If there is no clear consensus, just the first style should prevail.

I don't have a strong opinion on style, personally I basically imitate what there is in the code, but we are in a point that code grows without a clear control.

Reply:
If there is something done in non-test code that is there specifically as a test-hook I recommend adding a comment saying, "blah blah because it is a test hook".

Reply:
Ok, I added a comment